src/launch/cli/github/auth/commands.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+import click
+
+from launch.github.generate_github_token import get_token
+
+
+@click.command()
+@click.option(
+    "--application_id_parameter_name",
+    required=True,
```

Comment:
This is a good start with these options! We can take them a step or two further to make them even better.

First, I would recommend using the `type` argument to coerce these values to `int`. We know that the application and installation IDs should be numeric, we get that validation for free if we explicitly ask for an `int` when defining this option.

Next, and this is optional, you can define an environment variable that is automatically detected and plugged in so you don't have to specify the option at all if your environment contains everything that the script expects. Check out the `envvar` argument for more on this.

https://click.palletsprojects.com/en/8.1.x/options/#basic-value-options